File: lua/test/exists_test.lua
-- ProjectName SDK exists test

local sdk = require("eva-email-verification_sdk")

describe("EvaEmailVerificationSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)

File: rb/test/exists_test.rb
# EvaEmailVerification SDK exists test

require "minitest/autorun"
require_relative "../EvaEmailVerification_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = EvaEmailVerificationSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
